src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/FluentThemeState.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.ComponentModel;
+
+using System.ComponentModel.Design.Serialization;
+using System.Diagnostics;
+using System.Globalization;
+using System.Reflection;
+using System.Windows;
+using System.Windows.Markup;
+using System.Windows.Media;
+using System.Security;
+using MS.Internal;
+using MS.Utility;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Windows
+{
+    internal readonly struct FluentThemeState : IEquatable<FluentThemeState>
```

Comment:
This looks like an exemplary case for `record`

Reply:
Resolved this.